ui/lib/core/addon/components/form-field.hbs

```diff
@@ -152,17 +152,18 @@
               <RadioButton
                 class="radio"
                 name={{@attr.name}}
-                id={{or val.value val}}
-                value={{or val.value val}}
-                @value={{or val.value val}}
+                id={{or val.id (this.radioValue val)}}
```

Comment:
is adding `id` a new thing added by custom messages?? haven't seen those on radio button options before

Reply:
Ahhh yes good eye. I added that to the `possibleValues` object in the custom message form when I added the `radioValue`. Since the values are true and false it wasn't good for the id or the test selector to use those values so [I added an id to get around that](https://github.com/hashicorp/vault/pull/30238/files#diff-26ff3855051bbee2d3652a0f2de04dcac1303ae29f11a4cc0c3daa5970086f3aR27).